### PR TITLE
[2 of 2 System logs] Add new stream log view

### PR DESCRIPTION
**Pull request by mlunoe**
[System logs 1 of 2](https://github.com/dcos/dcos-ui/pull/1421)
[System logs 2 of 2](https://github.com/dcos/dcos-ui/pull/1422)

This PR introduces the view for the streaming log:

You need to start a cluster with `testing/pull/971` to test it out.

### CODE REVIEW

File: plugins/services/src/js/components/LogView.js [hunk 1]
```diff
@@ -0,0 +1,341 @@
+import React from 'react';
```
Comment (mlunoe) [inline plugins/services/src/js/components/LogView.js]: Most parts of this file are extracted from `MesosLogView`. There will be a follow-up task to extract the remaining parts to `TaskFileViewer` and make it use `LogView` instead, so we can remove `MesosLogView`.

File: plugins/services/src/js/pages/task-details/TaskDetail.js [hunk 2]
```diff
@@ -29,7 +30,9 @@ const HIDE_BREADCRUMBS = [
   '/jobs/:id/tasks/:taskID/details',
   '/network/virtual-networks/:overlayName/tasks/:taskID/details',
   '/nodes/:nodeID/tasks/:taskID/details',
-  '/services/overview/:id/tasks/:taskID/details'
+  '/services/overview/:id/tasks/:taskID/details',
+  '/services/overview/:id/tasks/:taskID/logs',
```
Comment (nLight) [inline plugins/services/src/js/pages/task-details/TaskDetail.js]: I think it should include optional params, like you specified further down
`logs(/:fileName)`

Comment (mlunoe) [inline plugins/services/src/js/pages/task-details/TaskDetail.js]: Arh! That's how it works!

File: plugins/services/src/js/components/LogView.js (hunk 1 above)
Comment (mlunoe) [inline plugins/services/src/js/components/LogView.js]: Will do!

Comment (mlunoe) [inline plugins/services/src/js/components/LogView.js]: Created! DCOS-11734

File: plugins/services/src/js/pages/task-details/TaskDetail.js (hunk 2 above)
Comment (mlunoe) [inline plugins/services/src/js/pages/task-details/TaskDetail.js]: Done

**Comment by mlunoe** [comment]
Also fixed an issue where the Files Tab would show up two times.

**Comment by mlunoe** [comment]
Fixed showing an error when logs are not supported

**Comment by mlunoe** [comment]
**mesosphere-ci** Restest this please

**Comment by mlunoe** [comment]
**mesosphere-ci** Restest this please

**Comment by jfurrow** [comment]
The Services and Deployments main-level tabs are displaying intermittently :(

**Comment by jfurrow** [comment]
Also the empty state seems like it should be centered, or maybe left-aligned?

**Comment by jfurrow** [comment]
I'm also able to get the log viewer in a state where it doesn't continually scroll down as new logs are received:

**Comment by mlunoe** [comment]
All comments addressed!

**Comment by jfurrow** [comment]
@mlunoe This is awesome, great work! It works nicely.

**Pull request merged**